flux/src/flagger/availabilitycheck.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState, useEffect } from 'react';
+import { K8s } from '@kinvolk/headlamp-plugin/lib';
+import { Loader } from '@kinvolk/headlamp-plugin/lib/components/common';
+import {Link, Box} from '@mui/material';
+export default function FlaggerAvailabilityCheck({ children }) {
+  const [isFlaggerInstalled, setIsFlaggerInstalled] = useState(null);
+  const [canary, error] = K8s.ResourceClasses.CustomResourceDefinition.useGet('canaries.flagger.app');
+  console.log("canary is ", canary);
+  console.log("error is ", error);
```

Comment:
If you need to print this, print it inside an effect so it's not constantly printing. And give it a better structure.

Reply:
updated